pages/version-policy/index.md

```diff
@@ -244,358 +234,242 @@ The designations are as follows:
+        <td><p style="text-align: center;">⚫</p></td>
     </tr>
     <tr>
-        <td>Jenkins 3.4.x-2.89.y</td>
+        <td>Jenkins 3.5.x-2.107.2</td>
         <td><p style="text-align: center;">⚫</p></td>
         <td><p style="text-align: center;">⚫</p></td>
         <td><p style="text-align: center;">⚫</p></td>
-        <td><p style="text-align: center;">⚫</p></td>
-        <td><p style="text-align: center;">◯</p></td>
     </tr>
     <tr>
-        <td>Kafka 1.x-*</td>
-        <td><p style="text-align: center;">◯</p></td>
-        <td><p style="text-align: center;">◯</p></td>
-        <td><p style="text-align: center;">◯</p></td>
-        <td><p style="text-align: center;">◯</p></td>
-        <td><p style="text-align: center;">◯</p></td>
+        <td>Jenkins 3.5.x-2.150.1 (Recommended)</td>
+        <td><p style="text-align: center;">⚫</p></td>
+        <td><p style="text-align: center;">⚫</p></td>
+        <td><p style="text-align: center;">⚫</p></td>
     </tr>
     <tr>
-        <td>Kafka 2.0.x-0.11.0</td>
-        <td><p style="text-align: center;">◯</p></td>
-        <td><p style="text-align: center;">◯</p></td>
-        <td><p style="text-align: center;">◒</p></td>
-        <td><p style="text-align: center;">◒</p></td>
-        <td><p style="text-align: center;">◯</p></td>
+        <td>Kafka 2.3.x-1.0.0</td>
+        <td><p style="text-align: center;">⚫</p></td>
+        <td><p style="text-align: center;">⚫</p></td>
+        <td><p style="text-align: center;">⚫</p></td>
     </tr>
     <tr>
-        <td>Kafka 2.0.x-1.0.0</td>
-        <td><p style="text-align: center;">◯</p></td>
-        <td><p style="text-align: center;">◯</p></td>
-        <td><p style="text-align: center;">◒</p></td>
-        <td><p style="text-align: center;">◒</p></td>
-        <td><p style="text-align: center;">◯</p></td>
+        <td>Kafka 2.3.x-1.1.0 (Recommended)</td>
+        <td><p style="text-align: center;">⚫</p></td>
+        <td><p style="text-align: center;">⚫</p></td>
+        <td><p style="text-align: center;">⚫</p></td>
     </tr>
     <tr>
-        <td>Kafka 2.1.x-1.0.0</td>
-        <td><p style="text-align: center;">⚫</p></td>
+        <td>Kafka-Zookeeper 2.3.x-3.4.12</td>
         <td><p style="text-align: center;">⚫</p></td>
         <td><p style="text-align: center;">⚫</p></td>
         <td><p style="text-align: center;">⚫</p></td>
-        <td><p style="text-align: center;">◯</p></td>
     </tr>
     <tr>
-        <td>Kafka-Zookeeper 2.0.x-3.4.11</td>
-        <td><p style="text-align: center;">⚫</p></td>
+        <td>Kafka-Zookeeper 2.4.x-3.4.13 (Recommended)</td>
         <td><p style="text-align: center;">⚫</p></td>
         <td><p style="text-align: center;">⚫</p></td>
         <td><p style="text-align: center;">⚫</p></td>
-        <td><p style="text-align: center;">◯</p></td>
     </tr>
     <tr>
-        <td>Kubernetes 1.0.x-*</td>
+        <td>Kubernetes 1.3.x-1.10.y</td>
         <td><p style="text-align: center;">⚫</p></td>
```

Comment:
Remove the one about these packages being supported on DC/OS 1.12. We have tested them for the purpose of migration but not for long term use.

Reply:
Done